__tests__/buildTranslationFiles.spec.ts

```diff
@@ -297,6 +298,42 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
         });
       });
     });
+
+    describe('scope', () => {
+      const type: TranslationCategory = 'scope';
+      const config = gConfig(type);
+
+      beforeEach(() => removeI18nFolder(type));
+
+      it('should work with scope', () => {
+        const scopes = [
+          'scope1',
+          'scope2',
+          'scope3',
+          'scope4',
+          'scope5',
+          'scope6',
+          'scope7',
+          'scope8',
+          'scope9',
+        ];
+
+        const expected = scopes.reduce(
+          (acc, scope) => ({ ...acc, [scope]: generateKeys({ end: 1 }) }),
+          {},
+        );
```

Comment:
I find this much more readable:
```suggestion
const expected = {};
for (const scope of scopes) {
  expected[scope] = generateKeys({ end: 1 });
}
```